app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -138,6 +138,19 @@ class SelectionInteractionViewModel private constructor(
         // TODO(#3624): Add warning to user when they exceed the number of allowable selections or are under the minimum
         //  number required.
         selectedItems += itemIndex
+        selectedItemText.set("You may select more choices")
+        updateIsAnswerAvailable()
+        true
+      }
+      selectedItems.size == 0 -> {
+        selectedItems += itemIndex
+        selectedItemText.set("Please select one or more choices.")
+        updateIsAnswerAvailable()
+        true
+      }
+      selectedItems.size == maxAllowableSelectionCount -> {
+        selectedItems += itemIndex
+        selectedItemText.set("No more than $maxAllowableSelectionCount choices may be selected.")
         updateIsAnswerAvailable()
         true
       }
```

Comment:
Create a separate function which accepts `size` and `maxAllowableSelectionCount ` which you can use to set the correct string.

Now decide where do want to use that function.

Reply:
Thanks, @rt4914 I have pushed the same, PTAL.